Sources/DeepLearning/Operators.swift

```diff
@@ -16,12 +16,12 @@
 import TensorFlow
 #endif
 
-// Rounds the values of a tensor to the nearest integer, element-wise.
+/// Rounds the values of a tensor to the nearest integer, element-wise.
```

Comment:
If you start with "rounds", it sounds like a mutating function, like [`FloatingPoint.round()`](https://developer.apple.com/documentation/swift/floatingpoint/2297801-round).

Instead, the comment should be like [`FloatingPoint.rounded()`](https://developer.apple.com/documentation/swift/floatingpoint/2295900-rounded).

Reply:
Thanks! Changed documentation to `Returns the values of the specified tensor to the nearest integer, element-wise.` in https://github.com/tensorflow/swift-apis/pull/46/commits/668c9be214a67683bc79ea8ac2fd577c5114bc32.